src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -27,6 +27,11 @@ public class PlayerTabList {
     public PlayerTabList(@NotNull Velocitab plugin) {
         this.plugin = plugin;
         this.players = new ConcurrentLinkedQueue<>();
+
+        // If the update time is set to 0 do not schedule the updater
+        if (plugin.getSettings().getUpdateRate() != 0) {
```

Comment:
```suggestion
        if (plugin.getSettings().getUpdateRate() > 0 {
```
this might still schedule the updater if the rate is set to a negative value (-1 is a fairly common sentinel) which will crash spectacularly (probably) -- so just probably want to do this check here

Reply:
fixed